DMFT/src/main/java/com/gov/rsi/DMFT/models/Reports.java

```diff
@@ -0,0 +1,56 @@
+package com.gov.rsi.DMFT.models;
+
+import org.bson.types.ObjectId;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document
+public class Reports {

```

Comment:
Is it a common java standard to name an class that represents a single object with a plural name?  I'd generally lean towards single `class Report`